solc/CommandLineInterface.cpp

```diff
@@ -1452,6 +1461,18 @@ bool CommandLineInterface::processInput()
 		m_modelCheckerSettings.engine = *engine;
 	}
 
+	if (m_args.count(g_argModelCheckerTargets))
+	{
+		string targetsStr = m_args[g_argModelCheckerTargets].as<string>();
+		optional<ModelCheckerTargets> targets = ModelCheckerTargets::fromString(targetsStr);
+		if (!targets)
+		{
+			serr() << "Invalid option for --" << g_argModelCheckerTargets << ": " << targetsStr << endl;
```

Comment:
I thought it would be nice to be more explicit. But I guess it is totally ok like it is.

Reply:
Yea, just basically followed the other error messages